cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=True)
+    description = serializers.CharField(max_length=255)
+    duration = serializers.IntegerField(required=True)
```

Comment:
required=True is a default value. False is not. If you want to make description not required you need to specify this in model as well. If you want to make description 255 char max you need to use Charfield for this field in model